iroh-net/Cargo.toml

```diff
@@ -86,6 +86,7 @@ tracing-subscriber = { version = "0.3", features = ["env-filter"], optional = tr
 
 # metrics
 iroh-metrics = { version = "0.11.0", path = "../iroh-metrics", default-features = false }
+strum = { version = "0.25.0", features = ["derive"] }
```

Comment:
I'm not a big fan of making sections in this list... because as shown here items end up in the wrong place.

Can you either remove the sections and sort alphabetically or move this to the right secion (still sorted alphabetically i guess).